moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js

```diff
@@ -144,6 +150,7 @@ const useColumns = ({
         field: "moped_proj_personnel_roles",
         width: 200,
         editable: true,
+        sortable: false,
```

Comment:
this is a cop out but with such a complicated renderCell function below, it just doesnt seem feasible to implement a custom sort here. do all columns really need to be sorted?

Reply:
not a cop out to me. thanks for stopping where it became complex. I am okay with this since the UI isn't showing a broken sort feature anymore. Thanks!

Reply:
i think this is fair!